docs/detections/add-exceptions.asciidoc

```diff
@@ -0,0 +1,241 @@
+=============
+
+[IMPORTANT]
+=====
+{ref}/binary.html[Binary fields] are not supported in detection rule exceptions.
+=====
+
+. Do one of the following:
++
+--
+
+* To add an Endpoint exception from the rule details page:
+.. Go to the rule details page (*Manage* -> *Rules*), and then search for and  select the Elastic *Endpoint Security* rule.
+.. Scroll down the rule details page, select the *Endpoint exceptions* tab, then click *Add endpoint exception*.
+
+* To add an Endpoint exception from the Alerts table:
+.. Go to *Alerts*.
+.. Scroll down to the Alerts table, and from an {elastic-endpoint}
+alert, click the *More actions* menu (*...*), then select *Add Endpoint exception*.
+
+* To add an Endpoint exception from Rule Exceptions page:
+.. Go to *Manage* -> *Rule Exceptions*.
+.. Expand the Endpoint Security Exception List or click the list name to open the list's details page. Next, click *Add endpoint exception*. 
++
+NOTE: The Endpoint Security Exception List is automatically created. By default, it's associated the Endpoint Security rule and any rules with the <<rule-ui-advanced-params, *{elastic-endpoint} exceptions*>> option selected.  
+
+--
++
+The *Add Endpoint Exception* flyout opens.
++
+[role="screenshot"]
+image::images/endpoint-add-exp.png[]
+
+. If required, modify the conditions.
++
+NOTE: Refer to <<ex-nested-conditions>> for more information on when nested conditions are required.
+
+. You can select any of the following:
+
+* *Close this alert*: Closes the alert when the exception is added. This option
+is only available when adding exceptions from the Alerts table.
+* *Close all alerts that match this exception and were generated by this rule*:
+Closes all alerts that match the exception's conditions.
+
+. Click *Add Endpoint Exception*. An exception is created for both the detection rule and the {elastic-endpoint}.
+
+[float]
+[[ex-nested-conditions]]
+=== Exceptions with nested conditions
+
+Some Endpoint objects contain nested fields, and the only way to ensure you are
+excluding the correct fields is with nested conditions. One example is the
+`process.Ext` object:
+
+[source, json]
+--------------------------------------------------
+{
+  "ancestry": [],
+  "code_signature": {
+    "trusted": true,
+    "subject_name": "LFC",
+    "exists": true,
+    "status": "trusted"
+  },
+  "user": "WDAGUtilityAccount",
+  "token": {
+    "elevation": true,
+    "integrity_level_name": "high",
+    "domain": "27FB305D-3838-4",
+    "user": "WDAGUtilityAccount",
+    "elevation_type": "default",
+    "sid": "S-1-5-21-2047949552-857980807-821054962-504"
+  }
+}
+--------------------------------------------------
+
+
+TIP: `code_signature.subject_name` refers to the process signature not the
```

Comment:
I know this tip was here before, but I'm trying to figure out how it connects to anything else on the page? No other mentions of `code_signature.subject_name`. Maybe cut it?

Reply:
I think this is mainly a tip about the example in lines 172-192. I'm ok with removing it though if it's not all that relevant to exceptions in general. :)